website/content/api-docs/secret/pki.mdx

```diff
@@ -1073,6 +1074,33 @@ $ curl \
 }
 ```
 
+### OCSP Request
+
+This endpoint retrieves an OCSP response (revocation status) for a given serial number. The request/response formats are
+based on [RFC 6960](https://datatracker.ietf.org/doc/html/rfc6960)
+
+At this time there are certain limitations of the OCSP implementation at this path.
+ 1. Only a single serial number within the request will appear in the response
+ 1. None of the extensions defined in the RFC are supported for requests or responses
+ 1. Ed25519 backed CA's are not supported for OCSP requests
+
+These are unauthenticated endpoints.
```

Comment:
Might be worthwhile to add a note saying this is not a typical Vault response and cannot be serviced via the Vault CLI?

Reply:
Added new line item for this as well.